admin/installer/_images.py

```diff
@@ -53,7 +52,7 @@ class AWS_REGIONS(Values):
 
 DEFAULT_IMAGE_BUCKET = u'clusterhq-installer-images'
 DEFAULT_BUILD_REGION = AWS_REGIONS.US_WEST_1
-DEFAULT_DISTRIBUTION = u"ubuntu-14.04"
+DEFAULT_DISTRIBUTION = u"ubuntu-16.04"
```

Comment:
Can we have the default just be the first element of DISTRIBUTIONS?
This way there's only one place to change.